app/views/games/show.html.erb

```diff
@@ -28,95 +63,96 @@
+          <div class="black left"><%= render_piece(7,4) %></div>
+        </div>
 
-      <div class="row">
-        <div class="black left"><%= render_piece(0,3) %></div>
-        <div class="white left"><%= render_piece(1,3) %></div>
-        <div class="black left"><%= render_piece(2,3) %></div>
-        <div class="white left"><%= render_piece(3,3) %></div>
-        <div class="black left"><%= render_piece(4,3) %></div>
-        <div class="white left"><%= render_piece(5,3) %></div>
-        <div class="black left"><%= render_piece(6,3) %></div>
-        <div class="white left"><%= render_piece(7,3) %></div>
-      </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,3) %></div>
+          <div class="white left"><%= render_piece(1,3) %></div>
+          <div class="black left"><%= render_piece(2,3) %></div>
+          <div class="white left"><%= render_piece(3,3) %></div>
+          <div class="black left"><%= render_piece(4,3) %></div>
+          <div class="white left"><%= render_piece(5,3) %></div>
+          <div class="black left"><%= render_piece(6,3) %></div>
+          <div class="white left"><%= render_piece(7,3) %></div>
+        </div>
 
-      <div class="row">
-        <div class="white left"><%= render_piece(0,2) %></div>
-        <div class="black left"><%= render_piece(1,2) %></div>
-        <div class="white left"><%= render_piece(2,2) %></div>
-        <div class="black left"><%= render_piece(3,2) %></div>
-        <div class="white left"><%= render_piece(4,2) %></div>
-        <div class="black left"><%= render_piece(5,2) %></div>
-        <div class="white left"><%= render_piece(6,2) %></div>
-        <div class="black left"><%= render_piece(7,2) %></div>
-      </div>
-      <div class="row">
-        <div class="black left"><%= render_piece(0,1) %></div>
-        <div class="white left"><%= render_piece(1,1) %></div>
-        <div class="black left"><%= render_piece(2,1) %></div>
-        <div class="white left"><%= render_piece(3,1) %></div>
-        <div class="black left"><%= render_piece(4,1) %></div>
-        <div class="white left"><%= render_piece(5,1) %></div>
-        <div class="black left"><%= render_piece(6,1) %></div>
-        <div class="white left"><%= render_piece(7,1) %></div>
-      </div>
+        <div class="row">
+          <div class="white left"><%= render_piece(0,2) %></div>
+          <div class="black left"><%= render_piece(1,2) %></div>
+          <div class="white left"><%= render_piece(2,2) %></div>
+          <div class="black left"><%= render_piece(3,2) %></div>
+          <div class="white left"><%= render_piece(4,2) %></div>
+          <div class="black left"><%= render_piece(5,2) %></div>
+          <div class="white left"><%= render_piece(6,2) %></div>
+          <div class="black left"><%= render_piece(7,2) %></div>
+        </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,1) %></div>
```

Comment:
Current setup is preventing you.

I think you have 2 options here.

1. Take step back. Start a new branch off of master that refactors the board so that each square isn't written out (and uses loops instead). Once that's done, continue working on ajax.
2. Use the current setup (hard code the positions in each DIV, super tedious, but will work)

Up to you!

Also don't feel any time pressure on this 👍 . I know this is a hard one. A lot of different pieces to learn at once. I recommending taking your time and focusing on each little piece at once. I find that makes it easier than trying to solve the the whole thing in one go.